### PR TITLE
Kill queue

### PR DESCRIPTION
**Pull request by Randium**
Aight, that's the whole kill queue thing, if I am not mistaken. Let's merge this.

### CODE REVIEW

File: test_main.py [hunk 1]
```diff
@@ -19,3 +19,16 @@ def test_positionof():
...
+import reset
+
+def kill_queue_test():
+  db.add_kill(12738912739821,"Barber")
+  db.add_kill(12347892374923,"White Werewolf",7289347983274)
+  print(db.get_kill())
+  print(db.get_kill())
+  print(db.get_kill())
+  reset.reset(True)
+  return True
```
Comment (Bentechy66) [inline test_main.py]: This is completely useless, the tests, by standards, shouldn't be printing anything. Use an `assert` statement to make sure `get_kill` is returning the right thing. You should also test that `reset.reset` works. I suggest making it return True if the reset is successful, and the error if else.

**Comment by Randium** [comment]
Eyyy! No problems with dr. Travis!

File: test_main.py [hunk 2]
```diff
@@ -19,3 +19,16 @@ def test_positionof():
...
+# Check if the kill_queue is fully functional
+import management.db as db
+import reset
+
+def kill_queue_test():
+  db.add_kill(12738912739821,"Barber")
+  db.add_kill(12347892374923,"White Werewolf","7289347983274")
+  assert db.get_kill() == [1,u'12738912739821',u'Barber',u'']
+  assert db.get_kill() == [2,u'12347892374923',u'White Werewolf',u'7289347983274']
+  assert db.get_kill() == None
```
Comment (Randium) [inline test_main.py]: Does this work?

Comment (Bentechy66) [reply test_main.py]: Looks good. Ok, changes approved, feel free to merge.

**Pull request merged**